Makefile

```diff
@@ -487,18 +491,37 @@ deploy: generate-examples
 	kubectl apply -f examples/_out/provider-components.yaml
 	kubectl apply -f examples/_out/metal3crds.yaml
 
+deploy-clusterclass: generate-examples-clusterclass
+	kubectl apply -f examples/_out/cert-manager.yaml
+	kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment cert-manager
+	kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment cert-manager-cainjector
+	kubectl wait --for=condition=Available --timeout=300s -n cert-manager deployment cert-manager-webhook
+	kubectl apply -f examples/_out/provider-components.yaml
+	kubectl apply -f examples/_out/metal3crds.yaml
+
 deploy-examples:
 	kubectl apply -f ./examples/_out/metal3plane.yaml
 	kubectl apply -f ./examples/_out/cluster.yaml
 	kubectl apply -f ./examples/_out/machinedeployment.yaml
 	kubectl apply -f ./examples/_out/controlplane.yaml
 
+deploy-examples-clusterclass:
+	kubectl apply -f ./examples/_out/metal3plane.yaml
+	kubectl apply -f ./examples/_out/clusterclass.yaml
+	kubectl apply -f ./examples/_out/cluster.yaml
+
 delete-examples:
 	kubectl delete -f ./examples/_out/machinedeployment.yaml || true
 	kubectl delete -f ./examples/_out/controlplane.yaml || true
 	kubectl delete -f ./examples/_out/cluster.yaml || true
 	kubectl delete -f ./examples/_out/metal3plane.yaml || true
 
+delete-examples-clusterclass:
+	kubectl delete -f ./examples/_out/cluster.yaml
```

Comment:
Don't you need to pass `|| true ` to all of these as in `delete-examples` above?

Reply:
Had no issues without those yet, but added those just in case.